### DISCUSSION TITLE
建议启动器可以设置仅在隧道启动失败时弹出状态提示

### DISCUSSION THREAD

**Discussion by NaTyphoon** [natfrp/wiki#124] [想法]
今天凌晨遇到API Failure导致隧道不启动了，直到下午4点多才发现……
由于隧道启动成功也会弹出状态提示，所以我平常一直是把状态提示设为关闭状态。但这就导致隧道启动失败也无法收到提示。
目前SakuraFrp启动器的状态提示设置只有简单的开关，建议多设一个选项：
- 个人希望SakuraFrp启动器能隧道启动成功不提示、启动失败（隧道退出且不自启）右下角弹出状态提示。

**Comment by SanaeAkahane** [comment]
![好！！！](https://user-images.githubusercontent.com/105238384/182021501-c0e5cd1f-4b78-47b6-bf2a-07c881f27923.jpg)
